x-pack/plugins/security_solution/public/resolver/view/resolver_without_providers.tsx

```diff
@@ -106,45 +106,47 @@ export const ResolverWithoutProviders = React.memo(
             </div>
           </div>
         ) : (
-          <GraphContainer
-            data-test-subj="resolver:graph"
-            className="resolver-graph kbn-resetFocusState"
-            onMouseDown={onMouseDown}
-            ref={ref}
-            role="tree"
-            tabIndex={0}
-            aria-activedescendant={activeDescendantId || undefined}
-          >
-            {connectingEdgeLineSegments.map(
-              ({ points: [startPosition, endPosition], metadata }) => (
-                <EdgeLine
-                  edgeLineMetadata={metadata}
-                  key={metadata.reactKey}
-                  startPosition={startPosition}
-                  endPosition={endPosition}
-                  projectionMatrix={projectionMatrix}
-                />
-              )
-            )}
-            {[...processNodePositions].map(([treeNode, position]) => {
-              const nodeID = nodeModel.nodeID(treeNode);
-              if (nodeID === undefined) {
-                throw new Error('Tried to render a node without an ID');
-              }
-              return (
-                <ProcessEventDot
-                  key={nodeID}
-                  nodeID={nodeID}
-                  position={position}
-                  projectionMatrix={projectionMatrix}
-                  node={treeNode}
-                  timeAtRender={timeAtRender}
-                />
-              );
-            })}
-          </GraphContainer>
+          <>
+            <GraphContainer
+              data-test-subj="resolver:graph"
+              className="resolver-graph kbn-resetFocusState"
+              onMouseDown={onMouseDown}
+              ref={ref}
+              role="tree"
+              tabIndex={0}
+              aria-activedescendant={activeDescendantId || undefined}
+            >
+              {connectingEdgeLineSegments.map(
+                ({ points: [startPosition, endPosition], metadata }) => (
+                  <EdgeLine
+                    edgeLineMetadata={metadata}
+                    key={metadata.reactKey}
+                    startPosition={startPosition}
+                    endPosition={endPosition}
+                    projectionMatrix={projectionMatrix}
+                  />
+                )
+              )}
+              {[...processNodePositions].map(([treeNode, position]) => {
+                const nodeID = nodeModel.nodeID(treeNode);
+                if (nodeID === undefined) {
+                  throw new Error('Tried to render a node without an ID');
+                }
+                return (
+                  <ProcessEventDot
+                    key={nodeID}
+                    nodeID={nodeID}
+                    position={position}
+                    projectionMatrix={projectionMatrix}
+                    node={treeNode}
+                    timeAtRender={timeAtRender}
+                  />
+                );
+              })}
+            </GraphContainer>
+            <PanelRouter />
```

Comment:
Realized there's no reason for this to ever load without the graph being present. Also, with Refresh you get states where the panel is trying to load data, but the tree hasn't even loaded yet.

Reply:
The panel can get back data and load before the graph, right? So why wouldn't the panel router need to be available immediately?

Reply:
Yep, was more of the assumption that the data without the context of the graph when in the analyze event tool wouldn't really make much sense, especially since you have access to that information in timeline apart from the graph. That being said, that wouldn't really have been much of a reason to move it into the ternary, but we saw some bugs with error states being triggered after refresh in the panel and decided to just move it in based on that for now. I'm pretty sure I've figured out what that bug is, but trying to figure out another bug with `panelViewAndParameters` not updating in some situations as well.

I can pull it back out if you think that's a better UX?